x-pack/legacy/plugins/lens/public/app_plugin/plugin.tsx

```diff
@@ -103,17 +121,27 @@ export class AppPlugin {
       return <FormattedMessage id="xpack.lens.app404" defaultMessage="404 Not Found" />;
     }
 
-    return (
-      <I18nProvider>
-        <HashRouter>
-          <Switch>
-            <Route exact path="/edit/:id" render={renderEditor} />
-            <Route exact path="/" render={renderEditor} />
-            <Route component={NotFound} />
-          </Switch>
-        </HashRouter>
-      </I18nProvider>
-    );
+    localApplicationService.register({
+      id: 'lens',
+      title: 'Lens',
```

Comment:
Can or should this be i18n?

Reply:
It's not really used right now, it's just there to have the same interface as the _real_ application service. I will use the `NOT_INTERNATIONALIZED_PRODUCT_NAME` for now.